i18n/en.json

```diff
@@ -3563,7 +3565,7 @@
   "user.settings.notifications.sensitiveUsername": "Your non-case sensitive username \"{username}\"",
   "user.settings.notifications.sensitiveWords": "Other non-case sensitive words, separated by commas:",
   "user.settings.notifications.soundConfig": "Please configure notification sounds in your browser settings",
-  "user.settings.notifications.sounds_info": "Notification sounds are available on IE11, Safari, Chrome and Mattermost Desktop Apps.",
+  "user.settings.notifications.sounds_info": "Notification sounds are available on Edge, Safari, Chrome and Mattermost Desktop Apps.",
```

Comment:
Not sure if the change to Edge is accurate but if it is, the default message where it's used should be updated too - https://github.com/mattermost/mattermost-webapp/blob/master/components/user_settings/notifications/desktop_notification_settings.jsx#L100.  Needs clarification for Dev reviewers

Reply:
Thanks. I fixed default message.

And I fixed this message because...
* IE11 supports was dropped in 5.16
* I tested in my local, Edge can play notification sound

Reply:
Thanks @kaakaa.  I'm all good, just need to update failing test snapshot.

Reply:
Oh... I've updated snapshot. Thanks!